src/libs/utils.ts

```diff
@@ -348,27 +360,34 @@ export const commaJoin = (list, conjunction = 'or') => {
   )(list))
 }
 
+// TODO: add good typing (remove any's)
 export const sha256 = async message => {
   const hashBuffer = await crypto.subtle.digest('SHA-256', new TextEncoder().encode(message))
   return _.flow(
-    _.map(v => v.toString(16).padStart(2, '0')),
+    _.map((v: any) => v.toString(16).padStart(2, '0')),
     _.join('')
   )(new Uint8Array(hashBuffer))
 }
 
-export const formatBytes = bytes => {
+export const formatBytes = (bytes: number): string => {
   if (bytes < 2 ** 10) {
     return `${bytes} B`
   }
 
-  const [prefix, divisor] = [
+  const lookup: [string, number][] = [
     ['P', 2 ** 50],
     ['T', 2 ** 40],
     ['G', 2 ** 30],
     ['M', 2 ** 20],
     ['K', 2 ** 10]
-  ].find(([_p, d]) => bytes >= d)
-  return `${(bytes / divisor).toPrecision(3)} ${prefix}iB`
+  ]
+  const maybeLookup = lookup.find(([_p, d]: [string, number]) => bytes >= d)
```

Comment:
this is to fix an unhandled edge-case which was exposed by Typescript complaining about this code's previous form.  But let me add some unit tests for this...

Reply:
unit tests added